fvm/transactionInvoker.go

```diff
@@ -87,6 +88,13 @@ func newTransactionExecutor(
 		ctx.EnvironmentParams,
 		txnState)
 
+	cadenceRuntime := env.BorrowCadenceRuntime()
+
```

Comment:
shouldn't you 
```
defer env.ReturnCadenceRuntime(cadenceRuntime)
```

Reply:
we do return it on the Cleanup call, this is mostly the setup phase